### PR TITLE
Add database requirements

### PR DESCRIPTION
**Pull request by liman4u**
**Description**
Add database requirements to challenge candidates on skills of reading and saving(creating) records in database.
For simplicity , sqlite was used and only has two tables.

### CODE REVIEW

**Review by bowiecao — APPROVED** [review]

File: db_connection.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  conn.commit()
+
+
+sqllite_conn = sql_connection()
+create_tables(sqllite_conn)
+populate_bounding_box(sqllite_conn)
+
+if (sqllite_conn):
+  sqllite_conn.close()
+  print("The SQLite connection is closed.")
```
Comment (katsgeorge) [inline db_connection.py:53]: Missing next line

**Comment by katsgeorge** [comment]
Could you elaborate on this choice? It feels a bit artificial, as we more often query DB for the data than storing something in it.

**Review by ghost — APPROVED** [review]

**Comment by katsgeorge** [comment]
🔥

File: README.md [hunk 2]
```diff
@@ -5,30 +5,31 @@ One of the ways the MI team evaluates a potential Ridepooling service is by simu
 
 
 ## Your task
-The Simulator itself is ready to go. Your mission is to integrate it in a solution that enables users to trigger a simulation and see its results visualized in some way accessible through a webpage.
+The Simulator itself is ready to go. Your mission is to integrate it in a solution that enables users to trigger a simulation and save part of the KPIs (booking distance bins) in database and see the simulation results visualized in some way accessible through a webpage.
```
Comment (nicolasiensen) [inline README.md]:
```suggestion
The Simulator itself is ready to go. Your mission is to integrate it in a solution that enables users to trigger a simulation and save part of the KPIs (booking distance bins) in database and see the simulation results in some way accessible through a webpage.
```

File: README.md [hunk 3]
```diff
@@ -5,30 +5,31 @@ One of the ways the MI team evaluates a potential Ridepooling service is by simu
...
-- **Testing the functionality of the Simulator itself.**
+- **Customized input to trigger the simulation(s).** It's okay if the number of request input is hardcoded or randomized. A simple button to trigger the simulation via the front-end is enough. It is expected, though, that any mocking would be done in the front-end layer.
+- **Testing the functionality of the Simulator.**
 
 
 ### Technical assumptions
-- The API/wrapper around the Simulator module should be built with Python3.
-- The front-end/webpage should be built with [React](https://reactjs.org/).
+- The API/wrapper around the Simulator module should be built with Python3 or preferred programming language.
+- The front-end/webpage should be built with [React](https://reactjs.org/) or any javascript alternative.
```
Comment (nicolasiensen) [inline README.md]: These two lines are quite confusing. I'm not sure if it is preferred to use Python3 and React or pick whatever language and front-end technology I want.

What do you think about listing the languages we will accept the code challenge, or say that the candidate can choose whatever they want without mentioning Python3 or React?

**Review by nicolasiensen — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (liman4u) [reply README.md]: Then we can only mention that we preferred to be done in Python 3 and React, what do you think?

Comment (nicolasiensen) [reply README.md]: Makes sense to me.

**Review by larasika — APPROVED** [review]
LGTM !

**Pull request merged**